symbulate/results.py

```diff
@@ -333,56 +358,106 @@ def plot(self, type=None, alpha=None, normalize=True, jitter=False,
             discrete_x = sum([(i > 1) for i in x_height]) > .8 * len(x_height)
             discrete_y = sum([(i > 1) for i in y_height]) > .8 * len(y_height)
 
+            discrete_both = discrete_x and discrete_y
+            continuous_both = not discrete_x and not discrete_y
+
             if type is None:
-                type = "scatter"
+                type = ("scatter",)
 
             if alpha is None:
                 alpha = .5
 
-            if type == "scatter":
+            if all(x in ("scatter") for x in type):
                 if jitter:
                     x += np.random.normal(loc=0, scale=.01 * (max(x) - min(x)), size=len(x))
                     y += np.random.normal(loc=0, scale=.01 * (max(y) - min(y)), size=len(y))
-                # get next color in cycle
-                axes = plt.gca()
-                color = get_next_color(axes)
-                plt.scatter(x, y, color=color, alpha=alpha, **kwargs)
-            elif type == "tile" and discrete_x and discrete_y:
+                nullfmt = NullFormatter() #removes labels on fig
+                fig, ax = plt.subplots(1, 1)
+                ax.scatter(x, y, alpha=alpha, c='b')
+            elif all(x in ("scatter", "marginal") for x in type):
+                if jitter:
+                    x += np.random.normal(loc=0, scale=.01 * (max(x) - min(x)), size=len(x))
+                    y += np.random.normal(loc=0, scale=.01 * (max(y) - min(y)), size=len(y))
+                nullfmt = NullFormatter() #removes labels on fig
+                fig = plt.figure()
+                gs = GridSpec(4, 4)
+                ax_joint = fig.add_subplot(gs[1:4, 0:3])
+                ax_marg_x = fig.add_subplot(gs[0, 0:3])
+                ax_marg_y = fig.add_subplot(gs[1:4, 3])
+                ax_joint.scatter(x, y, alpha=alpha)
+                ax_marg_x.hist(x)
+                ax_marg_y.hist(y, orientation='horizontal')
+                plt.setp(ax_marg_x.get_xticklabels(), visible=False)
```

Comment:
Is this different from `ax_marg_x.xaxis.set_ticklabels([])`?

Reply:
I thought that's how it was supposed to work when I making these plots, but it doesn't work that way. I'm sure why to be completely honest.